schema/postgresql.sql

```diff
@@ -22,7 +22,7 @@ CREATE TABLE clients (
 -- Store codes for use in Authorizatrion Code Grant
 -- https://tools.ietf.org/html/rfc6749#section-4.1
 CREATE TABLE request_codes (
-    code         UUID           NOT NULL DEFAULT uuid_generate_v4(),
+    code         VARCHAR(73)    NOT NULL DEFAULT uuid_generate_v4(),
```

Comment:
@oswynb not sure what you mean by ಠ_ಠ

I personally am not sure why 73, I'm pretty sure v4 uuids are 36 chars.